polar2grid/compare.py

```diff
@@ -302,6 +317,7 @@ def compare_dirs(self, dir1, dir2, **kwargs) -> list[FileComparisonResults]:
         return results
 
     def compare(self, input1, input2, **kwargs) -> list[FileComparisonResults]:
+        kwargs.update({"plot": self.create_plot})
```

Comment:
You are right that this is missing, but I think it might be cleaner if you move this to the `compare_files` method similar to how `self.margin_of_error` is used.

Reply:
Okay.  I will work to update this.